vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -53,6 +45,21 @@ IComponentView *CompositionBaseComponentView::parent() const noexcept {
   return m_parent;
 }
 
+bool CompositionBaseComponentView::runOnChildren(bool forward, Mso::Functor<bool(IComponentView &)> &fn) noexcept {
+  if (forward) {
+    for (auto it = m_children.begin(); it != m_children.end(); ++it) {
```

Comment:
I wonder if you could use algorithms.h here like std::for_each or something similar